### PR TITLE
Negative rebase deploy script

### PR DESCRIPTION
**Pull request by vp4242**
Negative rebase deploy script

### CODE REVIEW

File: scripts/scratch/neg-rebase-deploy.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+rm -f ${NETWORK_STATE_FILE}
+cp ${NETWORK_STATE_DEFAULTS_FILE} ${NETWORK_STATE_FILE}
+
+yarn compile
+
+rm -f ${NETWORK_STATE_FILE}
+cp ${NETWORK_STATE_DEFAULTS_FILE} ${NETWORK_STATE_FILE}
+
+
+yarn compile
```
Comment (tamtamchik) [inline scripts/scratch/neg-rebase-deploy.sh:52]: Why is that twice?

Comment (vp4242) [reply scripts/scratch/neg-rebase-deploy.sh:52]: That's probably a mistake. I would close this PR most probably as it's not needed anymore. Do you think it make sense to create a scratch deploy, specific for Sepolia? In that case this PR could be modified

Comment (tamtamchik) [reply scripts/scratch/neg-rebase-deploy.sh:52]: No, I just mentioned it in case this is still useful. I'll close the PR.

**Comment by tamtamchik** [comment]
Closed as it's not needed anymore.

**Pull request closed without merging**